plugins/Flanger/FlangerEffect.h

```diff
@@ -43,7 +44,10 @@ class FlangerEffect : public Effect
 	{
 		return &m_flangerControls;
 	}
+
+private slots:
 	void changeSampleRate();
+	void updatePhase();
 	void restartLFO();
```

Comment:
I figured it would be best to do it this way since it's modifying variables in PhaserEffect rather than PhaserControls.  Would it be better for me to put those functions into PhaserControls and modify the values in PhaserEffect from there?  I feel like that wouldn't look as neat.  What's the reason for keeping these things in PhaserControls?